app/controllers/merchants_controller.rb

```diff
@@ -46,5 +46,4 @@ def destroy
   def merchant_params
     params.permit(:name,:address,:city,:state,:zip)
```

Comment:
Maybe add spacing for convention / styling
` :name, :address, :city, :state, :zip`